src/main/java/no/nav/veilarbperson/client/pdl/PdlClientImpl.java

```diff
@@ -59,15 +60,21 @@ public PdlClientImpl(String pdlUrl, Supplier<String> systemUserTokenSupplier) {
         return graphqlRequest(request, userToken, HentPdlPerson.class).hentPerson;
     }
 
+    @Override
+    public HentPdlPerson.VergeOgFullmakt hentVergeOgFullmakt(String personIdent, String userToken) {
+        GqlRequest request = new GqlRequest<>(hentVergeOgFullmaktQuery, new PdlPersonVariables.HentPersonVariables(personIdent, false));
+        return graphqlRequest(request, userToken, HentPdlPerson.HentVergeOgFullmakt.class).hentPerson;
+    }
+
     @Override
     public HentPdlPerson.Familiemedlem hentPartner(String personIdent, String userToken) {
         GqlRequest request = new GqlRequest<>(hentPersonQuery, new PdlPersonVariables.HentPersonVariables(personIdent, false));
         return graphqlRequest(request, userToken, HentPdlPerson.Partner.class).hentPerson;
     }
 
     @Override
-    public List<HentPdlPerson.Barn> hentPersonBolk(String[] personIdent) {
-        GqlRequest request = new GqlRequest<>(hentPersonBolkQuery, new PdlPersonVariables.HentPersonBolkVariables(personIdent, false));
+    public List<HentPdlPerson.Barn> hentPersonBolk(String[] personsIdenter) {
```

Comment:
Litt småpirk, men et bedre navn er `personIdenter`. Hvis det står `personsIdenter` så gir det uttrykk for at det er 1 person som har flere identer